webpack.config.js

```diff
@@ -105,7 +105,7 @@ if (process.env.NODE_ENV === 'production') {
         scope: '/',
         events: true,
       },
-      externals: ['/zee-worker.js', '/worker.js', '/analytics.js'],
+      externals: ['/js/zee-worker.js', '/worker.js', '/js/analytics.js'],
```

Comment:
I don't think this should be changed, because `yarn build:clean` (https://github.com/devtools-html/perf.html/pull/957/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R7) still copies these files to the root of /dist. So the URLs will be `/zee-worker.js` and `/analytics.js`.

Alternatively you can copy these files to the /js subdirectory in `yarn build:clean`. _but_ in that case you'll need to change `src/utils/worker-factory.js` to go and find workers in that directory.